### DISCUSSION TITLE
Context Menu

### DISCUSSION THREAD

**Discussion by Woofman69** [kkrypt0nn/Python-Discord-Bot-Template#80] [Ideas]
Adding a cog that shows how to use context menu's with the template.

**Comment by kkrypt0nn** [comment]
Will be done in the next update

**Reply by kkrypt0nn** [reply]
Has now been [added](https://github.com/kkrypt0nn/Python-Discord-Bot-Template/blob/main/cogs/general.py#L22-L69)
